json-endpoints/class.wpcom-json-api-get-comments-tree-endpoint.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+class WPCOM_JSON_API_Get_Comments_Tree_Endpoint extends WPCOM_JSON_API_Endpoint {
+	/**
+	 * Retrieves a list of comment data for a given site.
+	 *
+	 * @param   {string} $status   filter by status: 'all' | 'approved' | 'pending' | 'spam' | 'trash'
+	 * @param   {int}    $start_at first comment to search from going back in time
+	 * @returns {array}            list of comment information matching query
+	 */
+	public function get_site_tree( $status, $start_at = PHP_INT_MAX ) {
+		global $wpdb;
+		$max_comment_count = 10000;
+		$total_count = $this->get_site_tree_total_count( $status );
+		$db_status = $this->get_comment_db_status( $status );
+
+		$db_comments = $wpdb->get_results(
```

Comment:
as the original author of the generic code that this was based on I can say that I found `db_comments` to be the most meaningful name. can you suggest an alternative?

the _meaning_ of `$db_comments` here is "rows from the comments table in the database" as distinct from full WordPress comment objects